numpy/lib/function_base.py

```diff
@@ -1755,6 +1757,10 @@ def disp(mesg, device=None, linefeed=True):
     '"Display" in a file\\n'
 
     """
+    # NumPy 1.17.0, 2019-04-04
+    warnings.warn(
+        "numpy.lib.function_base.disp is deprecated",
+        DeprecationWarning, stacklevel=2)
```

Comment:
This shows up when someone tries to use `disp`. The comment should end in gh-13214 (this issue). The text could mention `print` as an alternative